### ISSUE TITLE
Inkonsistens i Base URL for DAWA Service

### ISSUE THREAD

**Issue by kropsi** [SDFIdk/dawa-autocomplete2#15]
Hej,

Jeg har bemærket en inkonsistens i base URL for DAWA servicen. Dokumentationen peger på https://api.dataforsyningen.dk/ som base URL for servicen, men når [dawa-autocomplete2](https://github.com/SDFIdk/dawa-autocomplete2) anvendes, bruges https://dawa.aws.dk/ som base URL.

Dette skaber udfordringer, når vi anvender autocomplete-komponenten på blandt andet kommunernes hjemmesider, da vi er nødt til at whiteliste domæner, der må kaldes fra siden. I øjeblikket er vi derfor nødt til at whiteliste både https://dawa.aws.dk/ (som ikke er dokumenteret) og https://api.dataforsyningen.dk/.

Har I planer om at skifte https://dawa.aws.dk/ til https://api.dataforsyningen.dk/, så det følger dokumentationen? Dette vil også løse forklaringsproblemet ved whitelisting af API endpoints.

På forhånd tak for jeres hjælp!

**Comment by DocDemar** [comment]
Hej

Dataforsyningen understøtter en række legacy endpoints
Det skyldes ændringer i organisation og andre forandringer der er foregået gennem de 25 år Dataforsyningen har eksisteret.
Nogle af disse ændringer har givet anledning til at et nyt endpoint er tilføjet de eksisterende endpoints.

dawa.aws.dk er den nyeste tilføjelse. dawa.aws.dk blev tilføjet i forbindelse med insourcing af DAWA. Autocomplete er en del af DAWA og dermed blev denne komponent insourcet i Dataforsyningen og kunne derefter også tilgås med api.dataforsyningen.dk. 

Der er i praksis ingen forskel på om du benytte dawa.aws.dk eller api.dataforsyningen.dk
dawa.aws.dk og api.dataforsyningen.dk peger på den samme API gateway og understøtter identiske features.

De bedste hilsner
Jørgen Larsen de Martino

**Comment by kropsi** [comment]
@DocDemar ja, det er forståeligt, men så går jeg ud fra at det vil forsat kræve whitelisting af to domæner, som vi vil melde tilbage til kommunerne.
